styx-scheduler-service/src/main/java/com/spotify/styx/WorkflowCache.java

```diff
@@ -57,5 +43,5 @@
    *
    * @return an unordered set containing all Workflows
    */
-  ImmutableSet<Workflow> all();
```

Comment:
Consider removing the above `workflow(WorkflowId)` method as lookup can now be done using `all().get()`.

Reply:
Removed.

Reply:
Gone gone, like everything.